cxx/emissions/identity_test.cc

```diff
@@ -0,0 +1,31 @@
+// Apache License, Version 2.0, refer to LICENSE.txt
+
+#define BOOST_TEST_MODULE test Identity
+
+#include "emissions/identity.hh"
+
+#include <boost/test/included/unit_test.hpp>
+#include <random>
+
+BOOST_AUTO_TEST_CASE(test_simple) {
+  IdentityEmission<bool> ie;
+
+  BOOST_TEST(ie.logp_score() == 0.0);
+  BOOST_TEST(ie.N == 0);
+  ie.incorporate(std::make_pair<bool, bool>(true, false));
```

Comment:
Is it valid to incorporate a pair where `dirty != clean`? If not, could you make these pairs contain the same value, for self-documentation? (And consider adding an assertion in `incorporate`).

Reply:
It is not valid, so done.